src/logic/cookies/utils/index.ts

```diff
@@ -18,7 +19,7 @@ export const loadFromCookie = async (key) => {
   }
 }
 
-export const saveCookie = async (key, value, expirationDays) => {
+export const saveCookie = async (key: string, value: Record<string, any>, expirationDays: number): Promise<void> => {
```

Comment:
`any` -> `unknown`

Reply:
yes, I know, but if I set it as unknown it starts to break in other places and I didn't want to start fixing them in this PR.